### ISSUE TITLE
MagicFormat enum values change in 13.7.0

### ISSUE THREAD

**Issue by flytzen** [dlemstra/Magick.NET#1627]
### Magick.NET version

13.7.0

### Environment (Operating system, version and so on)

Any

### Description

The enum `Magick.Format` had a new value added in 13.7.0.
This can cause problems with nested dependencies and lead to an exception being thrown.

Specifically:

### Steps to Reproduce

ZXing is a library with bindings to a lot of image processors, including Magick. The `ZXing.Magick` binding specifically references `Magick.Net.Core` 12.3.0. 
It has a [line](https://github.com/micjahn/ZXing.Net/blob/45461597e3ef97ac00fb2b9f742612c9a69c8c0b/Source/Bindings/ZXing.Magick/MagickImageLuminanceSource.cs#L67) which specifically references `MagickFormat.Gray`. In v13.6.0 and before, that was `83`. However, in 13.7.0, `83` is `Gradient`.

Consumers of ZXing have to make an explicit reference to `Magick.Net` as `Magick.Net.Core` is not enough on its own. That, then, leads to Dependabot wanting to upgrade that reference to the latest, which then breaks `ZXing`.  Once a consumer of ZXing upgrades Magick.Net to 13.7.0, they will start getting exceptions about `Gradient` not being supported. This is because `ZXing` continues to pass in `83` to `Magick.Net` - but the meaning of that has changed.

The `GoogleAuthenticator` library I work on has run in to this: https://github.com/BrandonPotter/GoogleAuthenticator/pull/213

`ZXing` can of course be upgraded but that may cause other issues. 

You may or may not feel this is an issue in Magick.Net - I can see both arguments. I can see one way of resolving it in Magick.Net, which would be to explicitly assign integers to the `Magick.Format` enum - but that does risk breaking consumers of 13.7 :(

**Comment by dlemstra** [comment]
I don't know if version `12.3.0` of `Magick.Net.Core` is fully compatible with the latest version of ImageMagick. I have been added formats in alphabetical order for years but never realized this would be a breaking change. I can fix the changes I made for the version `13.X` range to allow backwards compatibility. But I will make a note that I will reorder them again for the `14.X` release. So when this release becomes available you will probably need to update the `Core` library. I had a release ready for today but I will include this patch to reorder the enums to allow backwards compatibility and publish a new release tonight.
